jdaviz/configs/imviz/plugins/tools.py

```diff
@@ -28,7 +44,7 @@ def activate(self):
 @viewer_tool
 class BqplotMatchPanZoom(BqplotPanZoomMode):
 
-    icon = os.path.join(ICON_DIR, 'panzoom_match.svg')
+    icon = os.path.join(ICON_DIR, 'panzoom_linked.svg')
```

Comment:
Why the renaming?

Reply:
probably just because there were two versions to consider at one point and I forgot to switch it back.  Fixed now.